models/host.go

```diff
@@ -116,6 +116,8 @@ const (
 	UpdateKeys HostMqAction = "UPDATE_KEYS"
 	// RequestPull - request a pull from a host
 	RequestPull HostMqAction = "REQ_PULL"
+	// JoinPull - request a pull from a host without restart
+	JoinPull HostMqAction = "JOIN_PULL"
```

Comment:
Can we change the name to 'SIGNAL_PULL'

Reply:
no problem.  Added a commit to update the name.    netclient change added as well.